core/src/test/java/org/springframework/security/authentication/DelegatingReactiveAuthenticationManagerTests.java

```diff
@@ -77,4 +77,36 @@ public void authenticateWhenBadCredentialsThenDelegate2NotInvokedAndError() {
 			.verify();
 	}
 
+	@Test
+	public void authenticateWhenDelegate1BadCredentialsThenDelegate2Authenticate() {
```

Comment:
Will you make sure these test names indicate that continueOnError is true? Maybe something like:

```java
public void authenticateWhenContinueOnErrorAndFirstBadCredentialsThenTriesSecond
```

Reply:
Done.